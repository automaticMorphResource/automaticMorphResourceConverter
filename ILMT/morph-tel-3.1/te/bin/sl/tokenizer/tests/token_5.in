ఆస్ట్రేలియా ఖండంలో ముందుగా చెప్పుకోవాల్సింది ప్రపంచంలోకెల్లా పెద్దదైన ఈ "పగడాల దిబ్బ" గురించే. 
