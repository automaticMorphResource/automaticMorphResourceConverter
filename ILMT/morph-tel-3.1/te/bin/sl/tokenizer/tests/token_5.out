<Sentence id="1">
1	ఆస్ట్రేలియా	unk
2	ఖండంలో	unk
3	ముందుగా	unk
4	చెప్పుకోవాల్సింది	unk
5	ప్రపంచంలోకెల్లా	unk
6	పెద్దదైన	unk
7	ఈ	unk
8	"	unk
9	పగడాల	unk
10	దిబ్బ	unk
11	"	unk
12	గురించే	unk
13	.	unk
</Sentence>

